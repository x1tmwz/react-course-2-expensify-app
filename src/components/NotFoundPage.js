import React from 'react';
import { Link } from "react-router-dom";

const NotFoundPage = () => (
    <div>
        <p>!404 -  <Link to="/">go home</Link></p> 
    </div>
);
export default NotFoundPage;