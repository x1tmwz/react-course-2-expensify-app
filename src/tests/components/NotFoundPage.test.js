import { shallow } from 'enzyme';
import React from 'react';
import NotFoundPage from '../../components/NotFoundPage';

test('should NotFoundPage Will render',()=>{
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot();
})