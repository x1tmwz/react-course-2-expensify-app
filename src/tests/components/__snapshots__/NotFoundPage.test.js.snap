// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should NotFoundPage Will render 1`] = `
<div>
  <p>
    !404 -  
    <Link
      to="/"
    >
      go home
    </Link>
  </p>
</div>
`;
